io.github.lpub3d.LPub3D.json

```diff
@@ -0,0 +1,183 @@
+{
+  "id": "io.github.lpub3d.LPub3D",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "lpub3d24",
+  "rename-icon": "lpub3d",
+  "rename-desktop-file": "lpub3d.desktop",
+  "rename-appdata-file": "io.github.lpub3d.appdata.xml",
+  "build-options": {
+    "env": {
+      "LP3D_3RD_PARTY_DIR": "$(realpath ./lpub3d_linux_3rdparty)",
```

Comment:
This definitely doesn't work:
1. shell substitution isn't performed here. Just variable.
2. I expect to resolve a path relative (inside) the build dir, but after building the module the build dir is deleted, which mean that using this for staging might not be a good idea.

Reply:
It’s hard to understand the logic of this platform. I only need the build dir through install, is it deleted before install is completed ? All my routines, including check, occur before install completion. I don’t understand why using this location, during the time that I will is not prudent.

Cheers,